tests/integration/local/invoke/test_invoke_terraform_applications.py

```diff
@@ -0,0 +1,92 @@
+import shutil
+from pathlib import Path
+from subprocess import Popen, PIPE, TimeoutExpired
+from typing import Optional
+from unittest import skipIf
+
+import pytest
+from parameterized import parameterized
+from pip._vendor.rich import json
```

Comment:
Why not just import json? Was this a autocomplete mistake?

Reply:
it is an auto complete mistake, I will update it

Reply:
Done